appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js

```diff
@@ -644,25 +644,38 @@ describe('appeal-details', () => {
 
 			expect(element.innerHTML).toMatchSnapshot();
 		});
-	});
 
-	it('should not render a back button', async () => {
-		const appealId = '2';
+		it('should not render a back button', async () => {
+			const appealId = '2';
 
-		nock('http://test/')
-			.get(`/appeals/${appealId}`)
-			.reply(200, {
-				...appealData,
-				appealId
-			});
+			nock('http://test/')
+				.get(`/appeals/${appealId}`)
+				.reply(200, {
+					...appealData,
+					appealId
+				});
+
+			const response = await request.get(`${baseUrl}/${appealId}`);
+			const element = parseHtml(response.text, { rootElement: 'body' });
+
+			const backButton = element?.querySelector('.govuk-back-link');
 
-		const response = await request.get(`${baseUrl}/${appealId}`);
-		const element = parseHtml(response.text, { rootElement: 'body' });
+			expect(backButton).toBeNull();
 
-		const backButton = element?.querySelector('.govuk-back-link');
+			expect(element.innerHTML).toMatchSnapshot();
+		});
 
-		expect(backButton).toBeNull();
+		it('should render a "Appeal valid" notification banner with a link to start case when status is "lpa questionnaire due"', async () => {
+			const appealId = 2;
+			nock('http://test/')
+				.get(`/appeals/${appealId}`)
+				.reply(200, { ...appealData, appealId, appealStatus: 'lpa_questionnaire_due' });
 
-		expect(element.innerHTML).toMatchSnapshot();
+			const response = await request.get(`${baseUrl}/${appealId}`);
+
+			expect(response.statusCode).toBe(200);
+			const element = parseHtml(response.text);
```

Comment:
The notification banners need to be handled differently to focus only on them.
You should be able to use the following: 
const element = parseHtml(response.text, { rootElement: '.govuk-notification-banner'})

Reply:
Also can you please check the element does have a link? 
expect(element.innerHTML).toContain('<a')